src/app/page/admin/decks/deck/manageCards/admin.manage.cards.component.html

```diff
@@ -1,7 +1,8 @@
 <div class="container">
   <div class="vertical-menu" *ngIf="deck">
-    <a [routerLink]="['/admin/manage/cards/', getDeckLink(deck.self)]" class="active">Delete/Update</a>
+    <a [routerLink]="['/admin/manage/cards/', getDeckLink(deck.self)]" class="active">Delete/Update card</a>
```

Comment:
I don't know how it looks on UI, but I suppose the message could be better. It is wrong to have 2 operations in one message. Something like 'Manage card' or 'Card operations' would be better.